### ISSUE TITLE
Scheduled calls for Protected Auction Services

### ISSUE THREAD

**Issue by pjl-google** [WICG/protected-auction-services-discussion#27]
We plan to hold a virtual meeting approximately every 2 weeks to discuss and resolve the details of the Trusted Servers that are part of Protected Audiences API and collect feedback from attendees.

The first meeting is planned for Wednesday December 13 2023 at noon New York (= 9am California = 6pm Paris time = 5pm UTC). Note that there is no meeting Dec 27 due to year-end holidays.  The second meeting will be on Wednesday January 3 2024.

The call will take place over Google Meet:
Video call link: https://meet.google.com/htk-obgg-smb 
Or dial: ‪(US) +1 509-676-3166‬ PIN: ‪117 463 428‬#
More phone numbers: https://tel.meet/htk-obgg-smb?pin=4553130257370

Agenda proposals, attendance records, notes will happen in this Google Doc.  Notes will be moved to Github after the meeting.
https://docs.google.com/document/d/1Hk6uW-i4KPUb-u20E-EWbu8_EbPnzcptnhV9mxBP5Mo/edit?usp=sharing 

If you want to participate in the call, please make sure you join the WICG: https://www.w3.org/community/wicg/

**Comment by pjl-google** [comment]
There will be an out-of-schedule meeting on Wednesday 20th December to discuss https://github.com/WICG/protected-auction-services-discussion/issues/31 and related topics.

This will use the same time and connection information as the regular meetings, above.

**Comment by thegreatfatzby** [comment]
Hey @pjl-google et al, can you guys do the thing MK has been doing where he edits the main issue with the next meeting date? I'm fairly sure the next one is next wednesday (17th) rather than in two days, but minor internal confusion here.

**Comment by pjl-google** [comment]
@thegreatfatzby certainly, that's a good idea, let me do that now.  And, yup, the 17th is the next scheduled one.

**Comment by thegreatfatzby** [comment]
Hey fwends, I believe our regularly scheduled meeting is tomorrow Wed April 10th, but PATCG F2F is happening...I do see [lunch](https://github.com/patcg/meetings/tree/main/2024/04/09-boston) is planned for that time so I guess it's not impossible but will we be holding this weeks?

**Comment by arinhad** [comment]
Hi everyone, we will be canceling tomorrow's WICG call for protected auction services and reconvene again in two weeks on 4/24.

**Comment by bmayd** [comment]
Is there a Google Group set up to sync calendar invites for this meeting series like the ones for PA API and ARA?

**Comment by yhkumargoogle** [comment]
Please note that the Aug 28, 2024 meeting has been canceled.

**Comment by yhkumargoogle** [comment]
Please note that the Sep 25, 2024 meeting has been canceled.

**Comment by bmayd** [comment]
> Is there a Google Group set up to sync calendar invites for this meeting series like the ones for PA API and ARA?

Bumping this -- is there a Google Group for scheduling like the Protected Audience API and Attribution Reporting API groups are using and if not, can someone create one?

**Comment by yhkumargoogle** [comment]
Tomorrow's (Oct 23) meeting has been canceled.

**Comment by yhkumargoogle** [comment]
Tomorrow's (Nov 20) meeting has been canceled. Next meeting: Dec 4

**Comment by dmdabbs** [comment]
@yhkumargoogle do you plan on having the call tomorrow, 18 Dec?

**Comment by yhkumargoogle** [comment]
We are canceling the meeting tomorrow (Dec 18) and on Jan 1 for the holidays. The next meeting is expected to be on Jan 15.

**Comment by yhkumargoogle** [comment]
Jan 15 call has been canceled as there were no topics noted for discussion. Next session: Jan 29

**Comment by yhkumargoogle** [comment]
Jan 29 call has been canceled as there were no topics noted for discussion. Next session: Feb 12

**Comment by samantha-jung** [comment]
March 26th, 2025 meeting has ben canceled due to lack of topics

**Comment by dmdabbs** [comment]
Assume no meeting 21 May?

**Comment by samantha-jung** [comment]
June 4th, 2025 meeting has been canceled due to lack of agenda

**Comment by samantha-jung** [comment]
June 18th, 2025 meeting has been canceled

**Comment by samantha-jung** [comment]
July 2nd, 2025 meeting has been canceled

**Comment by samantha-jung** [comment]
July 16th, 2025 meeting has been canceled

**Comment by bmayd** [comment]
@samantha-jung Given that the meeting is being canceled much more frequently than it is being held, it seems like we ought to cancel the series and instead of announcing cancelations here, announce when meetings get scheduled.

**Comment by samantha-jung** [comment]
Thanks for flagging this @bmayd , 

We are going to cancel this recurring call since there have been no suggested agenda items since 3/26/2025.
